i4trust_service.py

```diff
@@ -88,119 +87,260 @@ def on_post_product_spec_validation(self, provider, asset):
+
+            # Add supported post subscription
+            if asset.meta_info['post_subscription_allowed']:
+                self._append_string_charact(charact,
+                                            "POST Subscription Attributes Permission",
+                                            "Subscription attributes that can be posted (Notification attributes)",
+                                            asset.meta_info['post_subscription_attributes'])
+                self._append_string_charact(charact,
+                                            "POST Subscription IDs",
+                                            "Subscription IDs that can be posted",
+                                            asset.meta_info['post_subscription_ids'])
+
+            # Add supported delete subscription
+            if asset.meta_info['delete_subscription_allowed']:
+                self._append_string_charact(charact,
+                                            "DELETE Subscription Attributes Permission",
+                                            "Subscription attributes that can be deleted (Notification attributes)",
+                                            asset.meta_info['delete_subscription_attributes'])
+                self._append_string_charact(charact,
+                                            "DELETE Subscription IDs",
+                                            "Subscription IDs that can be deleted",
+                                            asset.meta_info['delete_subscription_ids'])
+
+            # Add notification type
+            if asset.meta_info['notification_allowed']:
+                self._append_string_charact(charact,
+                                            "Notification type",
+                                            "Entity type in notification",
+                                            asset.meta_info['notification_resource_type'])
+                self._append_string_charact(charact,
+                                            "Notification Attributes Permission",
+                                            "Entity attributes in notification",
+                                            asset.meta_info['notification_attributes'])
+                self._append_string_charact(charact,
+                                            "Notification Identifiers",
+                                            "IDs of corresponding subscriptions",
+                                            asset.meta_info['notification_ids'])
+
+            # Add Sidecar-Proxy endpoint config service
+            if asset.meta_info['sidecar_endpoint_config_allowed']:
+                self._append_string_charact(charact,
+                                            "Sidecar-Proxy endpoint configuration service",
+                                            "Access to sidecar-proxy endpoint configuration service allowed",
+                                            "True") # TODO: Can one use boolean instead?
```

Comment:
@fdelavega Is it possible to add characteristics of type boolean instead of string?

Reply:
Yes, it is possible

Reply:
What is the name of the function then?